adoc/retail_migr_chap_introduction.adoc

```diff
@@ -0,0 +1,26 @@
+[[retail_migr.chap.introduction]]
+= Introduction
+
+include::entities.adoc[]
+
+
+[[retail.sect.intro.what]]
```

Comment:
So if ``retail_migr`` is used as a book name, this should be ``[[retail_migr.sect.intro.what]]``, right?